src/plugins/condition/components/Condition.vue

```diff
@@ -68,13 +68,15 @@
                                         <input v-model="domainObject.configuration.name"
                                                class="t-condition-input__name"
                                                type="text"
+                                               :disabled="!telemetry.length"
```

Comment:
Hmm. Wondering if it matters that the user could change the condition name or the output value if there is no telemetry dragged into the conditionSet.
This would also prevent the default condition's name and output from being changed when the default condition is not dependent on any telemetry being present.

Reply:
Charles - please weigh in here on your view. My thought was it should act like SW where you can't edit the widget at all if no telemetry has been added.

Reply:
I was following the pattern in SW where the widget is not editable at all until telemetry is added. Charles - your thoughts?

Reply:
waiting for Charles' take

Reply:
Charles says to let the default conditions name and output be editable. Made the changes.